code/datums/wounds/_wounds.dm

```diff
@@ -72,13 +73,11 @@
 	var/limp_chance
 	/// How much we're contributing to this limb's bleed_rate
 	var/blood_flow
-	/// Essentially, keeps track of whether or not this wound is capable of bleeding (in case the owner has the NOBLOOD species trait)
-	var/no_bleeding = FALSE
 
-	/// The minimum we need to roll on [/obj/item/bodypart/proc/check_wounding] to begin suffering this wound, see check_wounding_mods() for more
-	var/threshold_minimum
 	/// How much having this wound will add to all future check_wounding() rolls on this limb, to allow progression to worse injuries with repeated damage
 	var/threshold_penalty
+	/// How much having this wound will add to all future check_wounding() rolls on this limb, but only for wounds of its own series
+	var/series_threshold_penalty = 0
```

Comment:
This is actually one of the least complex things I've added - this variable is used in precisely two locations: check_wounding() and check_series_wounding_mods(), and its probably one of the better additions. threshold_penalty exists as a way to let wounds progress with damage, but there are absolutely usecases where you only want that progression within the intended series of wounds. 

In fact I honestly might expand this further to let wounds decide if they want to only increase threshold for a certain damage type, a certain specific type, etc. but thats for later

Reply:
Also, with robotic wounds, it may be a good idea to make dislocations only give a series penalty, since a augmented limb can now get two different kinds of moderate blunt wounds, which will increase the woundability of the limb by a bit extra when it really shouldn't

Reply:
Hmmm, alright